README.md

```diff
@@ -220,6 +227,8 @@ Then, you will need to install PyTorch: refer to the [official installation page
 pip install accelerate
 ```
 
+
+
```

Comment:
No need for those two new lines here :-)

Reply:
Removed.